app/views/browse/config-maps.html

```diff
@@ -38,7 +38,7 @@ <h2>No config maps.</h2>
                 </p>
               </div>
               <div ng-if="filterWithZeroResults">
-                <h2>The filter is hiding all config maps. <a href="" ng-click="clearFilter()" role="button" class="nowrap">Clear Filter</a></h2>
+                <h2>The filter is hiding all config maps. <a href="" ng-click="clearFilter()" role="button" class="nowrap">Clear All Filters</a></h2>
```

Comment:
If we're changing all of these anyway, I suggest we go ahead and make them buttons for better accessibility.

```html
<button type="button" class="btn btn-link nowrap" ng-click="clearFilter()">Clear All Filters</button>
```

(We should double check if `nowrap` is needed with `btn-link`.)